database/Cassandra/schema.cql

```diff
@@ -0,0 +1,21 @@
+CREATE KEYSPACE mykeyspace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 2};
+
+CREATE TABLE songs (  
+   song_id int,
+   length int,
+   time_stamp int,
+   is_liked int,
+   song_data_url text,
+   song_name text,
+   artist text,
+   album text,
+   thumbnail_url text,
+   PRIMARY KEY (song_id)
+);
+
+CREATE TABLE likes (
+   id int,
+   song_id int,
+   user_id int,
```

Comment:
are you planning to make a users table? if not, it might make more sense for this to be `user` or `username` rather than an id

Reply:
I changed user_id to userName, because all I'll need here is if the current user has liked the current song